pkg/yurthub/storage/etcd/keycache.go

```diff
@@ -120,48 +120,49 @@ func (c *componentKeyCache) Recover() error {
 }
 
 func (c *componentKeyCache) getPoolScopedKeyset() (*keySet, error) {
-	client := c.getEtcdClient()
```

Comment:
It seems the function is uninitialized. I'll fix it.

Reply:
Will you fix it into my pr or create a new one ? @Congrool

Reply:
I'll create a new pr to fix it.

Reply:
fixed by #1137